MANIFEST.in

```diff
@@ -1,4 +1,6 @@
 include requirements.txt
 include requirements-test.txt
+include hamilton/server/requirements-mini.txt
+recursive-include hamilton/server/build *
```

Comment:
I'm confused, does this mean every Hamilton package will include the contents of build? That doesn't sounds right to me.

Reply:
Yeah it's not easy to disambiguate, although we could probably hack it in